src/pages/index.js

```diff
@@ -21,7 +24,8 @@ export default function Home() {
           ))}
         </section>
       </main>
-      <FooterBar />
+      < FooterBar />
```

Comment:
@Sheyitofunmi I have a strong suspicion that you are not updating your feature branches (like this current one named `aside-components`) with `main` branch, before you continue your work.

You MUST update with `main` every time you come back to work on your branch, because you have to ensure you have the most up-to-date version of `main` so that you don't end up duplicating or reversing work.

Case in point: this `<FooterBar>` component. I remember when I had asked you to re-add it, because you had removed it. But now, `main` branch has the `<FooterBar>` component in `index.js` already.

If I merge this branch into `main` as-is, you would have introduced a second `<FooterBar>` component into `index.js`.

So while on your `aside-components` branch, run the command `git pull origin main` to pull the most updated version of the app into your branch. Then resolve any conflicts that occur after the pull.

Let me know if I need to explain this further. I think I will also bring this up in Sprint Planning. Well done.